### ISSUE TITLE
Improve article 1904.00962

### ISSUE THREAD

**Issue by yuxiliu1995** [dginev/ar5iv#253]
**Exact location of issue**
Abstract

**Problem details**

Abstract is displayed as a blockquote.

![image](https://user-images.githubusercontent.com/33951560/195961395-2e77a19e-7aaf-4325-a1a2-e7d7335b170f.png)

**Desktop (please complete the following information)**
 - OS: Windows
 - Browser firefox chrome

**Comment by dginev** [comment]
Right, this would require some very peculiar context-sensitive CSS rules I think. 

The authors of that paper actually must have used the `{quote}` environment (or equivalent) to lay out their abstract, but in their PDF there would be no added decoration - while currently ar5iv adds this left sidebar to all quotes.

Explaining in case someone is wondering - I even recently gave using {quote} for abstracts as an example of messy markup in a talk.

**Comment by natfb** [comment]
Hello, I'm a first-timer in open source, what exactly would have to be done to solve this issue?

**Comment by dginev** [comment]
@natfb for this particular issue it would go something like:
 - open the article in question with the browser inspector and observe what markup was generated
 - a missing step is that we don't have exhaustive documentation for the HTML dialect we output, so you'd have to ask me or another latexml developer for class/element names that aren't obvious.
 - Then one would have to design a heuristic for which kinds of `blockquote` elements may be article abstracts. These may include:
    - immediately after the author list (`div.ltx_authors + `)
    - the quote is the only content of a paragraph (`.ltx_para > blockquote:only-child`)
    - (maybe?) the quote is not followed by an abstract
    - (maybe?) containing cite elements
       - some ideas are possible only once the new CSS `:has` is available in all browsers.
 - These ideas get put together in a larger CSS selector rule.
 - Ideally you test locally on a few articles - I think I can find one that starts with a real quote *and* has a proper abstract.

After all that, you'd likely want to file a pull request with the modified CSS to the [ar5iv-css](https://github.com/dginev/ar5iv-css/) repository. I'll help to nail down the details, merge and then deploy to the live ar5iv site, then closing the issue.

I think a workflow like that would make sense, right? It is specific to issues that only have to touch on CSS, and I think this one could be approached in that way for a quick upgrade.

**Comment by natfb** [comment]
@dginev Right, can I be assigned this issue?

**Comment by dginev** [comment]
All yours @natfb , thanks for volunteering!

**Comment by dginev** [comment]
Quick note on how I would set the work up on my machine @natfb , though of course you are welcome to work in whichever way you find convenient.

1. I'd fork [ar5iv-css](https://github.com/dginev/ar5iv-css/), clone it locally and switch to a new feature branch
   ```
    git checkout -b fix-ar5iv-issue-253
   ```
3. then I'd download the article in question, either via the browser or a quick
    ```
     $ wget --no-check-certificate https://ar5iv.labs.arxiv.org/html/1904.00962
    ```
4. Then switch to the local CSS with a replacement from:
    ```html
     <link media="all" rel="stylesheet" href="/assets/ar5iv.0.7.5.min.css">
    ```
   - to:
    ```html
     <link media="all" rel="stylesheet" href="css/ar5iv.css">
    ```

5. And edit the `css/ar5iv.css` file until results match expectations.
6. Lastly, file a new PR for ar5iv-css

**Comment by dginev** [comment]
This has been resolved with the PR linked above! 
(Actually, the specific paper was also updated to the latest version -- but ar5iv has guards for 3 different classes of this "quote-abstract" markup).

Closing and thanks again for the report @yuxiliu1995 .
